mkFit/MkFinder.cc

```diff
@@ -1309,6 +1309,17 @@ void MkFinder::BkFitFitTracksBH(const EventOfHits   & eventofhits,
                             Err[iP], Par[iP], msErr, msPar, Err[iC], Par[iC], tmp_chi2, N_proc);
     }
 
+    //fixup invpt sign and charge
```

Comment:
Shouldn't we put this in KalmanOperation[Endcap], if (update-requested)?
At least it should also go in BkFitFitTracks() as this is the one I'm switching to for backward search --- I'd have to remember to also put it in there.

Reply:
I couldn't quickly figure out how to pass a non-const charge